MobileCenterDistribute/MobileCenterDistribute.xcodeproj/project.pbxproj

```diff
@@ -901,9 +901,17 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				LIBRARY_SEARCH_PATHS = "\"$(SRCROOT)/../Vendor\"/**";
+				OTHER_LDFLAGS = (
```

Comment:
We do use config but this is platform specific. For `Foundation`, I have to double check, it is just copied from.